src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -288,6 +316,19 @@ private static boolean fieldOutOfRangeOrNotInteger(
     return !stringValid;
   }
 
+  private static boolean fieldIsDefinedButShouldNotBeForProvider(String value, String fieldName,
```

Comment:
I can't help wondering if there's a more concise name for this like fieldIsValidForProvider

Reply:
The only good alternative I could think of was `fieldUnnecessarilyDefinedForProvider`, which doesn't save that much space, so I'll go ahead and leave as-is.